### PR TITLE
Update `README.md` and `metadata.json` for `7.0.0-preview.3`

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

Update the metadata.json and README.md for 7.0.0-preview.3

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,21 +74,21 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 [rl-raspbian64]: https://github.com/PowerShell/PowerShell/releases/download/v6.2.2/powershell-6.2.2-linux-arm64.tar.gz
```
Comment (PoshChan) [inline README.md]: @adityapatwardhan, your last commit had 15 failures in `PowerShell-CI-static-analysis`
(These are 5 of the failures)

Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/PowerShell-7.0.0-preview.3-win-x64.msi should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/PowerShell-7.0.0-preview.3-win-x86.msi should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/powershell-preview_7.0.0-preview.3-1.ubuntu.18.04_amd64.deb should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/powershell-preview_7.0.0-preview.3-1.ubuntu.16.04_amd64.deb should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/powershell-preview_7.0.0-preview.3-1.debian.9_amd64.deb should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```

**Review by TravisEz13 — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@PoshChan please retry static

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started retry of `PowerShell-CI-static-analysis`

File: README.md (hunk 1 above)
Comment (PoshChan) [inline README.md]: @adityapatwardhan, your last commit had 16 failures in `PowerShell-CI-static-analysis`
(These are 5 of the failures)

Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/powershell-preview-7.0.0_preview.2-1.rhel.7.x86_64.rpm should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/PowerShell-7.0.0-preview.3-win-x64.msi should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/PowerShell-7.0.0-preview.3-win-x86.msi should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/powershell-preview_7.0.0-preview.3-1.ubuntu.18.04_amd64.deb should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-preview.3/powershell-preview_7.0.0-preview.3-1.ubuntu.16.04_amd64.deb should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 404 (Not Found).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**